### DISCUSSION TITLE
GJS.Market - you can now monetize your talent and sell your incredible plugins and presets to a global audience.

### DISCUSSION THREAD

**Discussion by GoodPHP** [GrapesJS/grapesjs#5202] [Ideas]
📢 Attention developers! 🚀 Exciting news from [https://gjs.market](https://gjs.market:) You can now monetize your talent and sell your incredible plugins and presets to a global audience. 🌍💡

🔥 Unlock new opportunities and showcase your skills on GJS.Market. Join today and start earning from your creations! 💰💻

👉 Create an account, understand the guidelines, and package your plugins/presets with care. Provide clear documentation for easy integration.

👉 List your products with captivating descriptions and visuals. Price competitively, and don't forget exceptional customer support!

👉 Market your offerings on social media, forums, and collaborate with fellow developers. Spread the word and watch your sales soar!

🌟 With GJS Marketplace's revenue-sharing model, you can build a steady income stream and expand your customer base. 💸💼

Don't miss out on this incredible opportunity to turn your passion into profit. Join [https://my.gjs.market/](https://my.gjs.market/) and start selling your plugins and presets today! 🚀💡

**Comment by ronaldohoch** [comment]
Nice work! I will think about create some UI blocks!

Also, i'm trying to figure out how do you make it! 😅
https://[devfuture.pro/live/placeholder2/](https://devfuture.pro/live/placeholder2/)

**Reply by GoodPHP** [reply]
We look forward to seeing you on our project with quality work for the community.

About the placeholder plugin, it's a commercial secret but for you we'll open little bit. We use the sorter events for it and some magic  😄
